content/50_Instrumenting_Lambda_Functions/51_Python/1_Add_Instrumentation.md

```diff
@@ -45,22 +45,22 @@ Finally, create a new variable called **AWS_LAMBDA_EXEC_WRAPPER** and set its va
 ![image](/images/instrumenting_lambda_functions/python/Python_Serverless_with_Vars.png)
 
 {{% notice note %}}
-The **AWS_LAMBDA_EXEC_WRAPPER** environment variable is only used for Python functions if the version of Python is 3.8. Versions 3.6 and 3.7 will use a different environment variable which is outlined in the [product documentation](https://docs.appdynamics.com/display/PRO45/Use+the+AppDynamics+AWS+Lambda+Extension+to+Instrument+Serverless+APM+at+Runtime).
+The **AWS_LAMBDA_EXEC_WRAPPER** environment variable is only used for Python functions if the version of Python is 3.8. Versions 3.6 and 3.7 will use a different environment variable which is outlined in the [product documentation](https://docs.appdynamics.com/21.6/en/application-monitoring/install-app-server-agents/serverless-apm-for-aws-lambda/use-the-appdynamics-aws-lambda-extension-to-instrument-serverless-apm-at-runtime).
 {{% /notice %}}
 
 Save your changes, and now we will add the AppDynamics AWS Lambda Extension to each of the Lambda functions. Locate the **functions** section within **serverless.yml**. Within that section will be 2 functions: *lambda-1* and *lambda-2*. To each of these functions, add the following:
 
 ``` yaml
     layers:
-      - arn:aws:lambda:${opt:region, self:provider.region}:716333212585:layer:appdynamics-lambda-extension:9
+      - arn:aws:lambda:${opt:region, self:provider.region}:716333212585:layer:appdynamics-lambda-extension:10
```

Comment:
The "appdynamics-lambda-extension:10" uses the updated versions of NPM and PyPI when bootstrapping the tracer. Please see more details here: 
https://docs.appdynamics.com/21.6/en/product-and-release-announcements/past-releases/past-agent-releases#PastAgentReleases-AppDynamicsAWSLambdaExtensionforServerlessAPM